i18n/ru/cosmic_bg.ftl

```diff
@@ -0,0 +1,2 @@
+app-name = Задние фоны Cosmic
```

Comment:
COSMIC should be capitalized

Reply:
In this case, EN and UA ftls may need to be changed as well (as they all use "Cosmic").

Should I add them to this PR?

Reply:
I updated "COSMIC" for all existing locales, however there are 3 PRs with new translations, so the same should either be requested for them, or updated after merging.